src/netbase.cpp

```diff
@@ -408,7 +408,7 @@ static bool Socks5(const std::string& strDest, int port, const ProxyCredentials
                 CloseSocket(hSocket);
                 return error("Error reading from proxy");
             }
-            int nRecv = pchRet3[0];
+            size_t nRecv = pchRet3[0];
```

Comment:
This look good, but I do not see this in the upstream PR. I am curious the upstream PR does not include this.

Reply:
It's probably not in the upstream PR to reduce the risk that someone would identify it as a covert fix of a security issue.